common/luaUtilities/color.lua

```diff
@@ -0,0 +1,30 @@
+local floor = math.floor
+local schar = string.char
+
+local function ColorStringEx(R, G, B, A, oR, oG, oB, oA)
+	-- Formats alpha and also outline color.
+	return "\254" .. schar(floor(R * 255)) .. schar(floor(G * 255)) ..
```

Comment:
```suggestion
	return Game.textColorCodes.ColorAndOutline .. schar(floor(R * 255)) .. schar(floor(G * 255)) ..
```

Reply:
Added it but linked through module local instead since I think that will be more efficient, please correct me if I'm wrong.

Reply:
Ok had to do like you said since Game is not really accessible out of the function :-P. I think it does make a small difference, probably around 5%... I did some benchmarks and inlining the floor(N*255) does seem to be better 
(10-15%) and more in line with code for ColorStringEx so I did change that.

In the end its not a really big difference... 10K StringColor calls seems to be about 7ms min with all optimizations and around 9ms without that's all (on a kind of potato pc).

Reply:
See https://github.com/beyond-all-reason/Beyond-All-Reason/pull/3877#issuecomment-2438201577

Reply:
ok, applied the recommendation from that comment and now works well. thanks!!